{"body": "All of the CI failures are due to an error like \"ReferenceError: SpecialPower is not defined\". Looks like you forgot the `s` on the end of <code>SpecialPower<strong>s</strong></code> somewhere.", "body_text": "All of the CI failures are due to an error like \"ReferenceError: SpecialPower is not defined\". Looks like you forgot the s on the end of SpecialPowers somewhere.", "url": "https://api.github.com/repos/mozilla/normandy/issues/comments/317797429", "created_at": "2017-07-25T16:43:13Z", "author_association": "MEMBER", "html_url": "https://github.com/mozilla/normandy/pull/895#issuecomment-317797429", "updated_at": "2017-07-25T16:43:13Z", "user": {"following_url": "https://api.github.com/users/mythmon/following{/other_user}", "events_url": "https://api.github.com/users/mythmon/events{/privacy}", "organizations_url": "https://api.github.com/users/mythmon/orgs", "url": "https://api.github.com/users/mythmon", "gists_url": "https://api.github.com/users/mythmon/gists{/gist_id}", "html_url": "https://github.com/mythmon", "subscriptions_url": "https://api.github.com/users/mythmon/subscriptions", "avatar_url": "https://avatars3.githubusercontent.com/u/305049?v=4", "repos_url": "https://api.github.com/users/mythmon/repos", "received_events_url": "https://api.github.com/users/mythmon/received_events", "gravatar_id": "", "starred_url": "https://api.github.com/users/mythmon/starred{/owner}{/repo}", "site_admin": false, "login": "mythmon", "type": "User", "id": 305049, "followers_url": "https://api.github.com/users/mythmon/followers"}, "body_html": "<p>All of the CI failures are due to an error like \"ReferenceError: SpecialPower is not defined\". Looks like you forgot the <code>s</code> on the end of <code>SpecialPower<strong>s</strong></code> somewhere.</p>", "id": 317797429, "issue_url": "https://api.github.com/repos/mozilla/normandy/issues/895"}